js/lib/components/IconDropdown.js

```diff
@@ -0,0 +1,71 @@
+        'social':
+            ["globe", 'amazon', 'angellist', 'apple', 'behance', 'bitbucket', 'codepen', 'connectdevelop', 'dashcube', 'delicious', 'deviantart', 'digg', 'dribbble', 'dropbox', 'drupal', 'facebook', 'flickr', 'foursquare', 'get-pocket', 'git', 'github', 'github-alt', 'gittip', 'google', 'google-plus', 'google-wallet', 'gratipay', 'hacker-news', 'instagram', 'ioxhost', 'joomla', 'jsfiddle', 'lastfm', 'leanpub', 'linkedin', 'meanpath', 'medium', 'odnoklassniki', 'opencart', 'pagelines', 'paypal', 'pied-piper-alt', 'pinterest-p', 'qq', 'reddit', 'renren', 'sellsy', 'share-alt', 'shirtsinbulk', 'simplybuilt', 'skyatlas', 'skype', 'slack', 'slideshare', 'soundcloud', 'spotify', 'stack-exchange', 'stack-overflow', 'steam', 'stumbleupon', 'tencent-weibo', 'trello', 'tripadvisor', 'tumblr', 'twitch', 'twitter', 'viacoin', 'vimeo', 'vine', 'vk', 'wechat', 'weibo', 'weixin', 'whatsapp', 'wordpress', 'xing', 'y-combinator', 'yelp', 'youtube-play' ],
```

Comment:
Please extract both lists to variables (put them at the beginning of the file, they're only scoped to this module anyway if you don't export them).
I'd prefer having one item per line - even if that gets very long, it's easier to see what's going on and diffs will be much easier to read.

Also, you don't need to wrap the keys in strings. :)